src/main.cpp

```diff
@@ -5146,6 +5162,19 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         }
     }
 
+    else if (strCommand == NetMsgType::FEEFILTER)
+    {
+        CAmount newFeeFilter = 0;
+        vRecv >> newFeeFilter;
+        if (MoneyRange(newFeeFilter)) {
+            {
+            LOCK(pfrom->cs_feeFilter);
```

Comment:
I remember adding the clang-format script some time ago. Something like `git diff -U0 HEAD~1.. | ./contrib/devtools/clang-format-diff.py -p1 -i -v` should do all of this.